.github/workflows/release.yml

```diff
@@ -0,0 +1,141 @@
+name: Release
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
+
+jobs:
+  Linux:
+    name: 'Build Linux Release'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get install ninja-build
+          sudo apt-get install gettext
+          pip3 install conan==1.36.0
+
+      - name: Conan common config
+        run: |
+          conan profile new --detect default
+          conan profile update settings.build_type=Release default
+          conan profile update settings.compiler.libcxx=libstdc++11 default
+
+      - name: Run Conan
+        run: |
+          mkdir build && cd build
+          conan profile list
+          conan profile show default
+          conan install .. -o webready=False --build missing
+
+      - name: Build packaged release
+        run: |
+          cd build
+          cmake -GNinja -DEXIV2_TEAM_PACKAGING=ON -DBUILD_SHARED_LIBS=ON -DEXIV2_ENABLE_WEBREADY=OFF -DEXIV2_ENABLE_NLS=ON -DEXIV2_BUILD_UNIT_TESTS=ON -DCMAKE_BUILD_TYPE=Release -DEXIV2_ENABLE_BMFF=ON -DEXIV2_TEAM_WARNINGS_AS_ERRORS=ON ..
+          cmake --build . -t package
+          tree -L 3
+
+      - uses: actions/upload-artifact@v2
+        with:
+          name: exiv2-linux64
+          path: ./build/exiv2-*.tar.gz
+          if-no-files-found: error
+          retention-days: 1
```

Comment:
Ah sorry I should have been clearer about this: 
My idea was to use these jobs to build the releases and add one more to the bottom of this file, which then downloads the artifacts, creates the GitHub release, and attaches the packages to the release. 

Thus, the files don't need a long retention since they are persisted in the release.